html/Mogo/js/fetchAPI/helper-functions.js

```diff
@@ -0,0 +1,30 @@
+function getMonthByValue(value) {
+  const months = {
+    Jan: 0,
+    Feb: 1,
+    Mar: 2,
+    Apr: 3,
+    May: 4,
+    Jun: 5,
+    Jul: 6,
+    Aug: 7,
+    Sep: 8,
+    Oct: 9,
+    Nov: 10,
+    Dec: 11,
+  };
+  return Object.keys(months).find((key) => months[key] === value);
+}
+
+function setPostsAttributes(attributes) {
+  const localAttributes = attributes;
```

Comment:
you just modify value, no reason for local variable, as here you just pass link to object which same this information, and this function looks a bit strange and date structure which it receive too